### ISSUE TITLE
Could not create `_$_TweetEditControls`. There is a problem with "edits_remaining".

### ISSUE THREAD

**Issue by myConsciousness** [twitter-dart/twitter-api-v2#571]
# 1. Current bug behavior

```bash
Unhandled exception:
CheckedFromJsonException
Could not create `_$_TweetEditControls`.
There is a problem with "edits_remaining".
type 'int' is not a subtype of type 'String' in type cast
#0      $checkedConvert (package:json_annotation/src/checked_helpers.dart:92:5)
#1      $checkedCreate.checkedConvert (package:json_annotation/src/checked_helpers.dart:29:7)
#2      _$$_TweetEditControlsFromJson.<anonymous closure> (package:twitter_api_v2/src/service/tweets/tweet_edit_controls.g.dart:17:42)
#3      $checkedCreate.<anonymous closure> (package:json_annotation/src/checked_helpers.dart:34:22)
#4      $checkedNew (package:json_annotation/src/checked_helpers.dart:52:23)
#5      $checkedCreate (package:json_annotation/src/checked_helpers.dart:31:10)
#6      _$$_TweetEditControlsFromJson (package:twitter_api_v2/src/service/tweets/tweet_edit_controls.g.dart:11:65)
#7      new _$_TweetEditControls.fromJson (package:twitter_api_v2/src/service/tweets/tweet_edit_controls.freezed.dart:157:7)
#8      _$TweetEditControlsFromJson (package:twitter_api_v2/src/service/tweets/tweet_edit_controls.freezed.dart:18:29)
#9      new TweetEditControls.fromJson (package:twitter_api_v2/src/service/tweets/tweet_edit_controls.dart:33:7)
```

# 2. Expected behavior

# 3. Steps to reproduce

# 4. Flutter doctor output

```terminal
Output of: flutter doctor -v
```

# 5. More environment information

# 6. Log information

```terminal
Enter log information in this code block
```

# 7. More information

**Comment by myConsciousness** [comment]
`@_RemainingCountConverter()` in `TweetEditControls` should be removed because something changed in Official Twitter API.

**Comment by myConsciousness** [comment]
Merged at #573
